### PR TITLE
add created/updated info to entity-card

### PR DESCRIPTION
**Pull request by mak7iliano**

### CODE REVIEW

File: packages/entity/src/entity-card-compact/entity-card-compact.component.html [hunk 1]
```diff
@@ -105,16 +105,39 @@ <h4>{{xmEntity.name}}</h4>
...
+            [xmEntityId]="xmEntity.id"
+            [xmEntitySpec]="xmEntitySpec"
+            [xmEntity]="xmEntity"
+            class="function-list">
+          </xm-function-list-section-compact>
+        </div>
+
+        <div class="col">
+          <div class="block-created-info">
+            <ng-container *ngIf="createdBy || xmEntity.startDate">
```
Comment (sergeysenja1992) [inline packages/entity/src/entity-card-compact/entity-card-compact.component.html:124]: xmEntity safe access

File: packages/entity/src/entity-card-compact/entity-card-compact.component.html [hunk 2]
```diff
@@ -105,16 +105,39 @@ <h4>{{xmEntity.name}}</h4>
...
+              Created
+              <ng-container *ngIf="createdBy">by {{updatedBy}}</ng-container>
```
Comment (sergeysenja1992) [inline packages/entity/src/entity-card-compact/entity-card-compact.component.html:126]: maybe copy-paste mistake?

File: packages/entity/src/entity-card-compact/entity-card-compact.component.html [hunk 3]
```diff
@@ -105,16 +105,39 @@ <h4>{{xmEntity.name}}</h4>
...
+              <ng-container *ngIf="!createdBy">date</ng-container>
+              {{xmEntity.startDate | date: 'MM/dd/yyyy hh:mm'}}
+              <br>
+            </ng-container>
+            <ng-container *ngIf="updatedBy || xmEntity.updateDate">
```
Comment (sergeysenja1992) [inline packages/entity/src/entity-card-compact/entity-card-compact.component.html:131]: safe access pls

File: packages/entity/src/entity-card/entity-card.component.ts [hunk 4]
```diff
@@ -17,13 +19,18 @@ export class EntityCardComponent implements OnInit {
 
     @Input() public xmEntity: XmEntity;
     @Input() public xmEntitySpec: XmEntitySpec;
+    @Input() public entityUiConfig: any;
```
Comment (sergeysenja1992) [inline packages/entity/src/entity-card/entity-card.component.ts:22]: pls use type

Comment (sergeysenja1992) [reply packages/entity/src/entity-card/entity-card.component.ts:22]: move this logic to child class

**Review by sergeysenja1992 — APPROVED** [review]

**Review by andrewkikot — APPROVED** [review]

**Pull request merged**